src/gql/provider.tsx

```diff
@@ -55,9 +56,43 @@ const Provider: React.FC = ({ children }) => {
           },
         },
       },
+      Query: {
+        fields: {
+          assets: {
+            keyArgs: ["sort", "pagination", ["first", "last"]],
```

Comment:
relayStylePagination was not useful?

Is it because the backend implementation does not follow the relayStylePagination? I think it is better to keep the implementation as close as possible to standards and commonly accepted concepts. By using commonly accepted concepts, you reduce the learning cost for someone seeing the code for the first time.

Of course, this does not apply if there is a reasonable cause.

Reply:
I wrote the reason in the PR description.

Reply:
Sorry I missed. OK I see.